tailwind.config.js

```diff
@@ -1,6 +1,7 @@
 const colors = require('tailwindcss/colors')
 
 module.exports = {
+    important: true,
```

Comment:
do we really need the "important" property in tailwind config since we want to use tailwind only for colors ? Colors that are defined and imported in Vuetify config (so I believe they are 'important' by default looking on a vuetify point of vue)
Because it means we will end up with 'important' everywhere (on vuetify and tailwind side)

Reply:
Yeah you may be right, especially since, when I want to add CSS I generally try to use Tailwind but those custom classes aren't marked as **!important** by this so...

Reply:
Removed.